Source/hydro/riemann_nd.F90

```diff
@@ -370,47 +369,55 @@ subroutine riemann_state(qm, qp, qpd_lo, qpd_hi, &
        ! new values for gamc and (rho e) on the edges that are
        ! thermodynamically consistent.
 
-       do j = jlo, jhi
-          do i = ilo, ihi
+       do k = lo(3), hi(3)
+          do j = lo(2), hi(2)
+             do i = lo(1), hi(1)
 
-             ! this is an initial guess for iterations, since we
-             ! can't be certain that temp is on interfaces
-             eos_state % T = 10000.0e0_rt
+                ! this is an initial guess for iterations, since we
+                ! can't be certain that temp is on interfaces
+                eos_state % T = 10000.0e0_rt
 
-             ! minus state
-             eos_state % rho = qm(i,j,kc,QRHO)
-             eos_state % p   = qm(i,j,kc,QPRES)
-             eos_state % e   = qm(i,j,kc,QREINT)/qm(i,j,kc,QRHO)
-             eos_state % xn  = qm(i,j,kc,QFS:QFS+nspec-1)
-             eos_state % aux = qm(i,j,kc,QFX:QFX+naux-1)
+                ! minus state
+                eos_state % rho = qm(i,j,k,QRHO)
+                eos_state % p   = qm(i,j,k,QPRES)
+                eos_state % e   = qm(i,j,k,QREINT)/qm(i,j,k,QRHO)
+                eos_state % xn  = qm(i,j,k,QFS:QFS+nspec-1)
+                eos_state % aux = qm(i,j,k,QFX:QFX+naux-1)
 
-             call eos(eos_input_re, eos_state)
+                call eos(eos_input_re, eos_state)
 
-             qm(i,j,kc,QREINT) = eos_state % e * eos_state % rho
-             qm(i,j,kc,QPRES)  = eos_state % p
-             !gamcm(i,j)        = eos_state % gam1
+                qm(i,j,k,QREINT) = eos_state % e * eos_state % rho
+                qm(i,j,k,QPRES)  = eos_state % p
+                !gamcm(i,j)        = eos_state % gam1
 
-          enddo
-       enddo
+             end do
+          end do
+       end do
 
-       ! plus state
-       do j = jlo, jhi
-          do i = ilo, ihi
+       do k = lo(3), hi(3)
+          do j = lo(2), hi(2)
+             do i = lo(1), hi(1)
 
-             eos_state % rho = qp(i,j,kc,QRHO)
-             eos_state % p   = qp(i,j,kc,QPRES)
-             eos_state % e   = qp(i,j,kc,QREINT)/qp(i,j,kc,QRHO)
-             eos_state % xn  = qp(i,j,kc,QFS:QFS+nspec-1)
-             eos_state % aux = qp(i,j,kc,QFX:QFX+naux-1)
+                ! this is an initial guess for iterations, since we
+                ! can't be certain that temp is on interfaces
+                eos_state % T = 10000.0e0_rt
```

Comment:
It looks like this was missing before -- does this change cause tolerance-level diffs?

Reply:
I don't think we had any tests that exercised this, but yes, it would change things to EOS tolerance.